src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -59,7 +59,7 @@
     <PackageReference Include="Microsoft.Azure.DurableTask.Redis" Version="0.1.1-alpha" />
     <PackageReference Include="Microsoft.Azure.DurableTask.Emulator" Version="2.1.3" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.4" />
-    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.3.1" />

```

Comment:
I checked to see that we aren't using a fixed version of this library in V1, and it looks like Functions doesn't have any dependencies on it, so we should be good to use whatever version we want. 

However, once we pick a version for Functions V1, do we have to keep that version consistent to avoid breaking customers?

Reply:
Thanks for checking this. I don't expect we'd need to freeze the version we're using. I'm not 100% sure, but my intuition is that V1 customers will have some flexibility to reference other 1.x.x versions. Even so, I'm slightly less worried about breaking customers since they would need to opt-into any updated versions anyways.